### ISSUE TITLE
Not working on Web

### ISSUE THREAD

**Issue by exonent** [rodydavis/easy_web_view#81]
Hi, I tried to use it on web but got an error:    `scheduleMicrotask(_applyFocusChange);`

My code: 
```
EasyWebView(
                  width: screenWidth.width,
                  height: screenWidth.height - headerHeight,
                  src:
                      'https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf',

                  //isHtml: false,
                  isMarkdown: false,
                  convertToWidgets: false,
                  // width: 100,
                  // height: 100,
                ),
```

On mobile works perfectly! But not in (Crhome)web.

My flutter doctor -v: 
```
flutter doctor -v
[✓] Flutter (Channel stable, 3.0.1, on macOS 12.4 21F79 darwin-x64, locale
    en-US)
    • Flutter version 3.0.1 at /Users/mpro/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision fb57da5f94 (3 weeks ago), 2022-05-19 15:50:29 -0700
    • Engine revision caaafc5604
    • Dart version 2.17.1
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/mpro/Library/Android/sdk
    • Platform android-31, build-tools 30.0.3
    • Java binary at:
      /Library/Java/JavaVirtualMachines/jdk-11.0.15.1.jdk/Contents/Home/bin/java
    • Java version Java(TM) SE Runtime Environment 18.9 (build
      11.0.15.1+2-LTS-10)
    • All Android licenses accepted.

This is taking an unexpectedly long time...[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] VS Code (version 1.67.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.42.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 12.4 21F79 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 102.0.5005.61

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by chillbrodev** [comment]
@rodydavis This is also a blocker for using this package on web. It works fine on mobile but it is not getting called on web. Could you take a look at this issue when you get a moment?

**Comment by giorell** [comment]
Were you able to fix this? I'm getting the same error.

**Comment by theRightHooprod** [comment]
Were you able to fix this? I'm getting the same error.

**Comment by DavidOrakpo** [comment]
This is still a very big issue. EasyWebView doesn't work for web anymore.

**Comment by Shiba-Kar** [comment]
Facing the same error
<img width="556" alt="Screenshot 2023-07-27 at 11 45 10 AM" src="https://github.com/rodydavis/easy_web_view/assets/24454722/5aac60c9-ea78-4ea3-828d-926bf85195d8">

**Comment by rodydavis** [comment]
Fixed on **2.0.0**!
